Fluent Interfaces/src/Application.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Application {
+	
+	private List<Document<Integer>> documents = new ArrayList<>();
+
+	public Application() {
```

Comment:
What is the most readable way of constructing, initializing and adding an object to a collection of objects?